### ISSUE TITLE
Optional namespaces

### ISSUE THREAD

**Issue by zenDzeeMods** [int19h/Bannerlord.CSharp.Scripting#2]
In older versions of the game, there is no namespace "TaleWorlds.ObjectSystem". Can you make it optional?

```
            var options = GetScriptOptions().WithImports(
                "System",
                "System.Collections.Generic",
                "System.IO",
                "System.Linq",
                "System.Text",
                "TaleWorlds",
                "TaleWorlds.CampaignSystem",
                "TaleWorlds.CampaignSystem.Actions",
                "TaleWorlds.Core",
                "TaleWorlds.Library",
                "TaleWorlds.MountAndBlade"
                //"TaleWorlds.ObjectSystem"  <<< throws an exception
            );
```

**Comment by int19h** [comment]
As discussed on NexusMods, this is fixed in v0.2.0 by making the namespace list configurable. I'm going to keep the default list as is, because I only playtest on the most recent stable & beta versions, but you can tweak it as needed for your configuration.

**Comment by int19h** [comment]
Fixed in https://github.com/int19h/csx/commit/0b4035d286848fe994c9d143b91dbaa6b5c3b68a
